### PR TITLE
User previously accepted permissions

### PR DESCRIPTION
**Pull request by greenpaw94**
Finished the story point.   LINK: https://www.pivotaltracker.com/story/show/161752748

Added comments and renamed some files to begin uniform naming conventions.

### CODE REVIEW

File: app/src/androidTest/java/com/example/bespinaf/a2d2/test_denies_permissions.java [hunk 1]
```diff
@@ -52,27 +44,20 @@ public void setUp(){
...
-
-    @Test
+    @Test // Makes sure that the Agree button is on the screen
     public void hasButton(){
         Button testButtonAgree = activity.findViewById(R.id.button_rules_agree);
         assertNotNull(testButtonAgree);
     }
 
-    @Test
+    @Test //Tests the user can see an error pop-up when they deny permissions
```
Comment (Medlinator) [inline app/src/androidTest/java/com/example/bespinaf/a2d2/test_denies_permissions.java]: Update comment to reflect standard practice. Example:

/**
 * Lorem ipsum
 *
 * @param derp
 */
public void String canUserAgree(String derp) {
}

**Review by Medlinator — CHANGES_REQUESTED** [review]
Updates comments to reflect standard practice. 

Suggestion: Implement Butterknife.

**Comment by daesthelos** [comment]
Amended styling and comments
Failed to integrate Butterknife: receiving dexcompilation error

**Comment by Medlinator** [comment]
Change toast to a snackbar.

**Pull request merged**